fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -1440,6 +1440,8 @@ void SQLiteDB::open(bool writable) {
 			TraceEvent("CreatedDBFile").detail("Filename", apath);
 		}
 	}
+	if (walFile.isError() && walFile.getError().code() != error_code_file_not_found)
```

Comment:
It is not clear to me why this is necessary. Is this because `dbFile.isError()` would also be true in that case? And if so, why not just check for `walFile.isError()` first?

Could you please add a short comment explaining why you did this?